.github/workflows/build_and_test.yaml

```
@@ -12,12 +12,12 @@ jobs:
   # StackStorm-Exchange/ci/.github/workflows/pack-build_and_test.yaml@master
 
   build_and_test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     name: 'Build and Test / Python ${{ matrix.python-version-short }}'
     strategy:
       matrix:
         include:
-          - python-version-short: "3.8"
+          - python-version-short: ["3.8", "3.9", "3.10", "3.11"]
             python-version: 3.8.17
             vault-version: "1.12.5-1"
             hvac-gh-tag: "v1.1.1"
```

This uses the latest version in the Ubuntu 22 image
```suggestion
          - python-version-short: "3.8"
            python-version: 3.8.18
            vault-version: "1.12.5-1"
            hvac-gh-tag: "v1.1.1"
          - python-version-short: "3.9"
            python-version: 3.9.21
            vault-version: "1.12.5-1"
            hvac-gh-tag: "v1.1.1"
          - python-version-short: "3.10"
            python-version: 3.10.16
            vault-version: "1.12.5-1"
            hvac-gh-tag: "v1.1.1"
          - python-version-short: "3.11"
            python-version: 3.11.11
            vault-version: "1.12.5-1"
            hvac-gh-tag: "v1.1.1"
```